lib/main.dart

```diff
@@ -321,14 +337,16 @@ class _TenTenOneState extends State<TenTenOneApp> {
           generatePaymentHistory(walletInfo.lightningHistory, walletInfo.bitcoinHistory);
         } else if (event is Event_ChannelState) {
           context.read<ChannelChangeNotifier>().update(event.field0);
+        } else if (event is Event_Init) {
+          tentenoneChangeNotifier.set(event.field0);
         } else {
           FLog.warning(text: "Received unexpected event: " + event.toString());
         }
       });
     } on FfiException catch (error) {
       FLog.error(text: "Failed to initialise: Error: " + error.message, exception: error);
     } catch (error) {
-      FLog.error(text: "Failed to initialise: Unknown error");
+      FLog.error(text: "Failed to initialise: Unknown error", exception: error);
```

Comment:
🔧 Saying that the error is unknown and then showing an exception is confusing.

Reply:
I guess `Unknown error` because its not an FfiException. This is a catch for any other error. Would you prefer removing the `: Unknown error` part from the message?

Reply:
> Would you prefer removing the : Unknown error part from the message?

Yep, I think it would be slightly clearer.